File: dayOne/python/numberGuesserApp.py
import random
count = 0;
        
while True: 
    number = random.randint(1, 100)
    attempts = 0;
    correctGuess = False;

    while attempts < 5:

        guess = int(input("Guess the number: "))

        if (guess < 1 or guess > 100): 

            System.out.print("Invalid number ")                    

        else:

            attempts += 1;

        if (guess == number):

            print("Correct");

            count += 1

            correctGuess = True;

            break;

        elif (guess < number):

            print("Too low")

        else:

            print("Too high")



        if not (correctGuess):

            print("Failed. The number was ", number );

            print("Score: ", count);

        response = input("Care to go another round??? (yes/no): ");

        if not (response == "yes"):

            print("Thanks for playing! Final score: ", count);

        if(attempts == 1):

            print("Legendary!!!");

        elif(attempts == 2):

            print("Excellent");

        elif(attempts > 2 and attempts < 5): 

            print("Good");

        elif(attempts == 5): 

            print("Close");

        else:

            print("Better luck");



    break;

        
        
    

File: dayOne/java/numberGuesserApp.java
import java.util.Scanner;
import java.util.Random;

public class NumberGuesserApp {

    public static void main(String[] args) {
        Scanner userInput = new Scanner(System.in);
        Random randomNumber = new Random();

        int count = 0;
        
        while (true) {
            int number = randomNumber.nextInt(100) + 1;
            int attempts = 0;
            boolean correctGuess = false;

            while (attempts < 5) {

                System.out.print("Guess the number: ");

                int guess = userInput.nextInt();

                if (guess < 1 || guess > 100) {

                    System.out.print("Invalid number ");                    

                }else{

                     attempts++;


                }

                if (guess == number) {

                    System.out.println("Correct");

                    count++;

                    correctGuess = true;

                    break;

                } else if (guess < number) {

                    System.out.println("Too low");

                } else {

                    System.out.println("Too high");
                }
            }

            if (!correctGuess) {

                System.out.println("Failed. The number was " + number );
            }

            System.out.println("Score: " + count);

            //userInput.nextLine();
            
            System.out.print("Care to go another round??? (yes/no): ");

            String response = userInput.nextLine().trim().toLowerCase();

            if (!response.equals("yes")) {

                System.out.println("Thanks for playing! Final score: " + count);

                if(attempts == 1) {
        
                    System.out.print("Legendary!!!");

                }else if(attempts == 2) {

                    System.out.print("Excellent");
            
                }else if(attempts > 2 && attempts < 5) {

                    System.out.print("Good");
            
               }else if(attempts == 5) {

                    System.out.print("Close");

               }else{

                    System.out.print("Better luck");
               
               }

                break;
            }
        }
        
    }
}
